conda-store-server/tests/test_auth.py

```diff
@@ -140,36 +141,191 @@ def test_authorization(conda_store, entity_bindings, arn, permissions, authorize
     assert authorized == authorization.authorize(entity, arn, permissions)
 
 
-def test_end_to_end_auth_flow(conda_store):
+@pytest.mark.parametrize("role_mappings_version", [1, 2])
+def test_end_to_end_auth_flow_v1(conda_store_server, testclient, authenticate, role_mappings_version):
+    # Configures authentication
+    namespace = f"this-{uuid.uuid4()}"
+    other_namespace = f"other-{uuid.uuid4()}"
+
+    conda_store = conda_store_server.conda_store
+
     authentication = AuthenticationBackend()
     authentication.secret = "supersecret"
 
     token = authentication.encrypt_token(
         AuthenticationToken(
-            primary_namespace="default",
-            role_bindings={
-                "default/*": ["viewer"],
-                "e*/e*": ["admin"],
-            },
+            primary_namespace=namespace,
+            # No default roles
+            role_bindings={},
         )
     )
 
     token_model = authentication.authenticate(token)
 
     authorization = RBACAuthorizationBackend(
```

Comment:
Pulling all this auth stuff that is setup for the auth tests into a helper function would help readability of these tests imo. This won't block approval, but would help maintainability I think.